integration/users/usersinfo_test.go

```diff
@@ -0,0 +1,426 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package users
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+
+	"github.com/FerretDB/FerretDB/integration"
+	"github.com/FerretDB/FerretDB/integration/setup"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/util/testutil"
+)
+
+func TestUsersinfo(t *testing.T) {
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t)
+	client := collection.Database().Client()
+
+	// Map of users per database suffix
+	dbToUsers := map[string][]bson.D{
+		"": {
+			{{"createUser", "one"}, {"roles", bson.A{}}, {"pwd", "pwd1"}},
```

Comment:
I think we could extract common parts like `{"roles", bson.A{}}` into the test code to make test cases simpler